docs/MainSite/Main.md

```diff
@@ -2,16 +2,20 @@
 Distributed under the MIT License.
 See LICENSE.txt for details.
 \endcond
-\mainpage Documentation Overview
+\mainpage Introduction
 
 <div class="toc">
 Table of Contents
 <ul>
-<li class="level1"><a href="#intro_sec">Introduction </a></li>
-<li class="level1"><a href="installation.html">Installation </a></li>
-<li class="level1"><a href="installation_on_clusters.html">Installation On Clusters</a></li>
-<li class="level1"><a href="doc_coverage/index.html">Documentation Coverage </a></li>
-<li class="level1"><a href="unit-test-coverage/index.html">Unit Test Coverage </a></li>
+<li class="level1"><a href="#intro_sec">Introduction</a></li>
```

Comment:
what is changing on these lines and why?

Reply:
This is the table of contents on the landing page. It just links to the same pages as the dropdown menu does (kind of a shortcut). I moved the docs/test coverage links down a bit, but wanted to ask if we should put these links somewhere else.

Reply:
I think having them there is fine, I don't know of a better location